sklearn/_isotonic.pyx

```diff
@@ -90,13 +88,14 @@ def _make_unique(np.ndarray[dtype=floating] X,
     cdef floating current_weight = 0
     cdef floating y_old = 0
     cdef int i = 0
-    cdef int current_count = 0
     cdef int j
     cdef floating x
     cdef int n_samples = len(X)
+    cdef floating eps = np.finfo(X.dtype).eps
```

Comment:
Actually I think the following would make more sense in restrospect:

```suggestion
    cdef floating eps = np.finfo(X.dtype).resolution
```

This should not change much but the definition of `resolution` feels closer to what we would intuitively expect.

Reply:
Are you happy to leave the var name as `eps` ?

Reply:
yes I am happy :)